backend/src/main/java/com/redhat/cloud/notifications/routers/InternalService.java

```diff
@@ -282,6 +298,80 @@ public void httpRoot() {
         .onFailure().recoverWithItem(throwable -> Response.status(Response.Status.BAD_REQUEST).entity(new RenderEmailTemplateResponse.Error(throwable.getMessage())).build());
     }
 
+    @POST
+    @Path("/defaultBehaviorGroups")
```

Comment:
`/defaultBehaviorGroups` or `/behaviorGroups/default`? Just asking, I don't have a strong preference.

Reply:
I didn't want to cause a conflict with the `/behaviorGroups/{something}` paths that's why I used that one - no strong preference either.

Reply:
`InternalService` does not contain any `/behaviorGroups/{something}`, they are all in `NotificationService` so the conflict risk is low. I have a slight preference for `/behaviorGroups/default` but I can live with `/defaultBehaviorGroups`. I'll let you choose 😄

Reply:
You're right.
I don't have a preference in the name and can update it.
 Actually I think the "default" name is confusing for what it really does, but not really worried as is an internal api